CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 # Unreleased
 
+- Add `ocaml.server.construct.chainConstruct` setting to configure construct chaining. (#1673)
```

Comment:
I think we could have a new group to distinguish options that are used to configure the server and options that are used to configure the client's commands behavior. Maybe `ocaml.commands.construct.recursiveCalls`

Reply:
@PizieDust did you forget this one ? After that I think the PR is ready to be merged :-)

Reply:
Oh my bad. let me fix that